app/config/locales/np.continents.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+return [
+    'AF' => 'अफ्रीका ',
+    'AN' => 'एन्टार्टिका ',
+    'AS' => 'एशिया',
+    'EU' => 'युरोप',
+    'NA' => 'उत्तर अमेरिका ',
+    'OC' => 'ओसिआनिया',
+    'SA' => 'दक्षिण  अमेरिका ',
```

Comment:
remove the white spaces between

Reply:
Please check issue description #623. I posted there succesfully.